choccy_chip/src/emulator/opcode.rs

```diff
@@ -975,4 +991,22 @@ mod tests {
         assert_eq!(emu.program_counter(), 18);
         assert_eq!(emu.get_register_val(0xf), 1); // now f is 1 since we overflowed
     }
+
+    #[test]
```

Comment:
you make a good case, I think we should set up some way we can emulate a press. This should be done when we are ready to implement the main loop.